docs/how-to/opentelemetry.md

```diff
@@ -0,0 +1,174 @@
+(opentelemetry)=
+# Observability and Instrumentation with OpenTelemetry and Jina
+
+```{hint}
+Prometheus only based metrics collection will be deprecated soon. Refer to {ref}`Monitor with Prometheus and Grafana <monitoring>` for the old setup.
```

Comment:
I might be wrong, but I think it should be "Prometheus-only" with the `-`. @alexcg1 can you help me on this one?

Reply:
Yup, hyphen needed